### DISCUSSION TITLE
[Release Vote] Vote for Release v0.5.0-rc2

### DISCUSSION THREAD

**Discussion by Aireed** [apache/amoro#1824] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Arctic 0.5.0 release.

The commit ID is https://github.com/NetEase/amoro/commit/6c26147b037dc3210b48517d3d14f341b91e0c0a

This corresponds to the tag: v0.5.0-rc2
https://github.com/NetEase/amoro/commits/v0.5.0-rc2
https://github.com/NetEase/amoro/tree/v0.5.0-rc2
Please check out, build, and test.

[ ] +1 Release this as Arctic 0.5.0
[ ] +0
[ ] -1 Do not release this because...

Please vote in the next 72 hours and add your test steps when voting.

**Comment by zhoujinsong** [comment]
+1

For testing the release in my local environment with the following step:

- Ran build and test with JDK8 and JDK17
- Ran the quick start guide for Iceberg format

**Comment by xxubai** [comment]
+1 
1. Build with JDK8 and hadoop2
2. Deploy it in our lab enviroment
3. Do some optimizations for native Iceberg format

**Comment by wangtaohz** [comment]
+1

1.Build with JDK8 and hadoop3
2.Ran quick start follow with flink 1.15.4 and `iceberg-flink-runtime-1.15-1.3.0.jar` for Iceberg Format and Mixed Format Table

**Comment by Aireed** [comment]
+1

1.Build with JDK8 and hadoop2
2.Ran quick start follow with flink 1.14.5 and iceberg-flink-runtime-1.14-1.1.0.jar for Iceberg Format and Mixed Iceberg Format Table

**Comment by shidayang** [comment]
+1
1. trino query is ok

**Comment by HuangFru** [comment]
+1
Ran docker quick start, and it's OK
